### ISSUE TITLE
Why not make a sticker pack!

### ISSUE THREAD

**Issue by falconmick** [troyhunt/hibp-stickers#1]
Hey Troy!

Over the years you've had some awesome logos + you've been involved in some pretty cool things! Imagine a sticker pack with Asafaweb sticker!!

I would be happy to contribute some time to building a Sticker purchasing portal powered by serverless, netlify and paypal (your accounts) provided somebody made the sticker designs and found the corresponding layouts on sticker mule.

But what would I ask in return? Simple :) donate all proceeds after serverless costs + paypal's fees + PnH to the charity of your choice. Maybe [Girls Who Code](https://girlswhocode.com/) or [Starlight](https://starlight.org.au/)?

**Comment by falconmick** [comment]
An example of this would be Wes Bos's sticker packs he's made in the past: https://bos.af/

Basically he makes a whole bunch of cool stickers that people might want to put on their laptop/whatever and you bundle em up and sell em for a fixed price around the world.

**Comment by the-wazz** [comment]
@falconmick FWIW, Asafaweb is now end-of-life. https://www.troyhunt.com/its-end-of-life-for-asafaweb/. And, your idea is not really an issue.

**Comment by falconmick** [comment]
Fully aware :)

**Comment by troyhunt** [comment]
Might be something for the future, right now I’m happy I’ve just been able to get *something*mup there!

**Comment by falconmick** [comment]
Yeah 100% I did some initial research and surprisingly the amount of work
required to do the logistics side is crazy so it would need to be better
thought out and handled after Christmas
